kamon-akka-http/src/main/scala/kamon/akka/http/instrumentation/ServerFlowWrapper.scala

```diff
@@ -146,6 +146,18 @@ object ServerFlowWrapper {
     }
   }
 
+  def async(interface: String, port: Int)(handler: HttpRequest => Future[HttpResponse])(implicit mat: Materializer): HttpRequest => Future[HttpResponse] = {
+    val wrappedFlow = apply(Flow[HttpRequest].mapAsync(1)(handler), interface, port)
+
+    val responses =
+      Source.queue[HttpRequest](10, OverflowStrategy.fail).async.via(wrappedFlow)
```

Comment:
I'm not really sure about the effect of this particular `10` here :thinking:

Reply:
yeah, that was quite an arbitrarily chosen number...